examples/jsm/loaders/NRRDLoader.js

```diff
@@ -318,7 +318,7 @@ NRRDLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 		parseHeader( _header );
 
 		var _data = _bytes.subarray( _data_start ); // the data without header
-		if ( headerObject.encoding === 'gzip' || headerObject.encoding === 'gz' ) {
+		if ( headerObject.encoding === 'gzip' || headerObject.encoding === 'gz' || headerObject.encoding === 'gzi' ) {
```

Comment:
Just for my understanding: `gzi` is just a different name for `gzip`, right? It does not refer to a special compression/decompression algorithm.

Reply:
Also, seems like we could just do this?

```js
headerObject.encoding.substring( 0, 2 ) === 'gz'
```